game-core/src/main/java/games/strategy/triplea/attachments/UnitSupportAttachment.java

```diff
@@ -301,7 +314,7 @@ public boolean getOffence() {
     return offence;
   }
 
-  public String getBonusType() {
+  public Tuple<Integer, String> getBonusType() {
```

Comment:
I'd encourage using lombok here, in that case there would be no need to update the getter.

Reply:
Updated.